api/src/main/java/org/hawkular/inventory/api/Inventory.java

```diff
@@ -26,4 +34,83 @@
 public interface Inventory extends AutoCloseable {
 
     Tenants.ReadWrite tenants();
+
+    /**
+     * Quickly navigate to the provided tenant.
+     *
+     * @param tenant the tenant to steer to.
+     * @return the access interface to the tenant
+     * @throws EntityNotFoundException if the tenant is not found
+     */
+    default Tenants.Single steerTo(Tenant tenant) throws EntityNotFoundException {
```

Comment:
Well, default methods have 2 uses.
1. the one you mention - adding new methods to ifaces in a backwards compatible manner
2. providing default behavior (essentially replacing abstract base impls - if the default behavior is stateless). See for example [here](https://github.com/tinkerpop/tinkerpop3/blob/master/gremlin-core/src/main/java/com/tinkerpop/gremlin/structure/Element.java#L49).

These are obviously the second use case.

If you really don't want them there though, I would not make these methods part of this interface at all, because they are essentially helper methods. Instead I'd add a utility class that would offer an API like:
```
Steer.in(inventory).to(environment)...
```